<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InboxWidget</class>
 <widget class="QWidget" name="InboxWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>691</width>
     <height>541</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="inboxLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item>
     <widget class="QLabel" name="inboxHeaderLabel">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(224, 255, 248);
font: 16pt &quot;Times New Roman&quot;;
color: rgb(107, 107, 107);</string>
      </property>
      <property name="text">
       <string>You're chatting with ...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="inboxScrollArea">
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="inboxWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>687</width>
         <height>514</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="inboxContentLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetNoConstraint</enum>
        </property>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>689</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="senderLayout">
    <item>
     <widget class="QLineEdit" name="messageEdit">
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>140</number>
      </property>
      <property name="placeholderText">
       <string>Hello, who is there?</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="messageLengthLabel">
      <property name="text">
       <string>00/60</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="sendButton">
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="resetButton">
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
